### ISSUE TITLE
[BUG] clicks with Promise.all don't work

### ISSUE THREAD

**Issue by mxschmitt** [microsoft/playwright#22339]
```ts
import { chromium } from 'playwright';

(async () => {
  const browser = await chromium.launch();
  const context = await browser.newContext();
  const page = await context.newPage();

  await page.setContent(`
    <div id="a1" onclick="console.log('a1')">a1</div>
    <div id="b1" onclick="console.log('b1')">b1</div>
  `);

  page.on('console', msg => console.log('PAGE LOG:', msg.text()));

  await Promise.all([page.locator('#a1').click(), page.locator('#b1').click()]);

  await page.waitForTimeout(5_000);

  await context.close();
  await browser.close();
})();
```

Expected: a1 and b1
Actual: only b1

**Comment by mxschmitt** [comment]
Closing as a won't fix. Clicks should always happen after each other and not in parallel.
